commands/alpha/rpkg/pull/command_test.go

```diff
@@ -121,13 +130,22 @@ data:
 apiVersion: config.kubernetes.io/v1
 kind: ResourceList
 items:
+- apiVersion: kpt.dev/v1
+  kind: KptRevisionMetaData
```

Comment:
I think using the KRM format probably make sense here and is aligned with what we have already done in kpt. But to stay consistent with the naming in Kubernetes, we should probably call this `KptRevisionMetadata`.

Reply:
Sure will update the name